### PR TITLE
[networkdirect] Add new port (only test, no merge)

### PR DESCRIPTION
**Pull request by MVoz**

### CODE REVIEW

**Comment by Voskrese** [comment]
пффф....

Cannot find path 'C:\Program Files (x86)\Windows Kits\10\Vsix\VS2019\WDK.vsix' because it does not exist.

**Comment by Voskrese** [comment]
домучил) like broke a)))

**Comment by Voskrese** [comment]
@BillyONeal 

`WDK repair + WDK.vsix + /p:SpectreMitigation=false` build ok

**Review by BillyONeal — CHANGES_REQUESTED** [review]
This needs to not install persistent things on the VM that affect other PRs.

File: scripts/azure-pipelines/windows/initialize-component.ps1 [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Write-Host "Installing Visual Studio 2019 WDK extension..."
+$vsPath = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\Enterprise"
+if (-not (Test-Path $vsPath)) {
+    $vsPath = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\Community"
+}
+if (-not (Test-Path $vsPath)) {
+  return
+}
+#Start-Process "$vsPath\Common7\IDE\VSIXInstaller.exe" "/q /a `"${env:ProgramFiles(x86)}\Windows Kits\10\Vsix\VS2019\WDK.vsix`"" -Wait
```
Comment (BillyONeal) [inline scripts/azure-pipelines/windows/initialize-component.ps1:27]: Clearly the vsix is not important because you commented that out

**Comment by BillyONeal** [comment]
Right, the spectre thing is why I didn't want to add the WDK to the test fleet last time this came up.

(I know you were mostly attempting a test here; I marked 'waiting' just so that someone doesn't come by, not read the discussion, and merge without looking too hard)

File: scripts/azure-pipelines/windows/initialize-component.ps1 (hunk 1 above)
Comment (Voskrese) [reply scripts/azure-pipelines/windows/initialize-component.ps1:27]: @BillyONeal 
no, was an error that the library was needed

this parameter helped `/p:SpectreMitigation=false`

**Comment by Voskrese** [comment]
@BillyONeal 
here I also need to explain? why do I want to add WDK and why am I testing for SpectreMitigation workarounds
what is the reason for suspending code testing?

**Comment by BillyONeal** [comment]
No, it's clear here that you're just adding a port. :)

**Comment by Voskrese** [comment]
OK, if you block someone else's attempts to succeed in overcoming port import problems.
wait until the fear disappears and make the MS-MPI port itself

> ок, раз блокируешь чужие попытки добиться успеха в преодолении проблем импорта портов.
> подождем когда исчезнет страх и сделаешь сам порт MS-MPI

looking forward to

**Comment by BillyONeal** [comment]
You don't need to explain anything here. I marked this as 'request changes' so that someone else doesn't accidentally merge this PR while the deploy scripts don't match the actual VMs.

Even though I have 'request changes' set you can still test all you want :)

**Comment by BillyONeal** [comment]
I'm not sure why it doesn't seem to be picking up your test run....

**Comment by BillyONeal** [comment]
Ah, that's why, sorry :)

**Pull request closed without merging**